lib/grim/image_magick_processor.rb

```diff
@@ -48,9 +48,18 @@ def save(pdf, index, path, options)
         command_env['PATH'] = "#{File.dirname(@ghostscript_path)}#{File::PATH_SEPARATOR}#{ENV['PATH']}"
       end
 
-      result, status = Open3.capture2e(command_env, command.join(' '))
+      log(command_env, command)
+      result, status = Open3.capture2e(command_env, command.join(" "))
 
       status.success? || raise(UnprocessablePage, result)
     end
+
+    def log(command_env, command)
+      Grim.logger.debug "Running imagemagick command"
```

Comment:
Probably not a big deal, but debug without a block does actually create the strings and such even if they aren't written to the log. Sometimes I do Grim.logger.debug { some expensive to generate string } as the block form lazily instantiates the string. Most likely these strings are easy to generate, so it shouldn't be a huge deal, but I thought I'd mention it.